vpcinfo.go

```diff
@@ -106,7 +107,41 @@ type Zone string
 // String returns z as a string value, satisfies the fmt.Stringer interface.
 func (z Zone) String() string { return string(z) }
 
-func whereAmI() (Platform, error) {
+func zoneFromSubnets(subnets []Subnet) (Zone, error) {
+	log.Println("NOTICE vpcinfo - getting zone from subnets")
```

Comment:
Are you particularly attached to these log lines?

Reply:
Nope, got rid of them.